Bug: Build fails on auto-upgrade-cronjob-k8s-image Make target

### Describe the bug.

Make returns error:
```
$ make auto-upgrade-cronjob-k8s-package
Producing Agent Auto Upgrade CronJob K8S docker image hyc-edge-team-nightly-docker-local.artifactory.swg-devops.com/ppc64el_auto-upgrade-cronjob_k8s:testing
unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /home/jenkins/workspace/NightlyBuilds/anax/anax-in-k8s/Dockerfile.auto-upgrade-cron.ppc64el: no such file or directory
make: *** [Makefile:445: auto-upgrade-cronjob-k8s-image] Error 1
```

### Describe the steps to reproduce the behavior.

_No response_

### Expected behavior.

_No response_

### Screenshots.

_No response_

### Operating Environment

Cloud

### Additional Information

_No response_

--- a/Makefile
+++ b/Makefile
@@ -443,7 +443,7 @@ anax-k8s-image: anax-k8s-clean
 auto-upgrade-cronjob-k8s-image: auto-upgrade-cronjob-k8s-clean
 	@echo "Producing Agent Auto Upgrade CronJob K8S docker image $(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_STG)"
 	if [[ $(arch) == "amd64" || $(arch) == "ppc64el" || $(arch) == "arm64" ]]; then \
-		cd $(ANAX_K8S_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_LABELS) -t $(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_STG) -f Dockerfile.auto-upgrade-cron.$(arch) .; \
+		cd $(ANAX_K8S_CONTAINER_DIR) && docker build $(DOCKER_MAYBE_CACHE) $(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_LABELS) -t $(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_STG) -f Dockerfile.ubi.auto-upgrade-cron.$(arch) .; \
 	fi
 	docker tag $(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_STG) $(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_BASE):$(CRONJOB_AUTO_UPGRADE_K8S_IMAGE_VERSION)
 

